source/routing/raptor_api.cpp

```diff
@@ -572,7 +573,7 @@ static bt::ptime handle_pt_sections(pbnavitia::Journey* pb_journey,
                 const auto p_deptime = item.departures[i];
                 const auto p_arrtime = item.arrivals[i];
                 pb_creator.action_period = bt::time_period(p_deptime, p_arrtime + bt::seconds(1));
-                pb_creator.fill(item.stop_points[i], stop_time->mutable_stop_point(), 0);
+                pb_creator.fill(item.stop_points[i], stop_time->mutable_stop_point(), depth - 1);
```

Comment:
if `depth==0` we will get some strange results

Reply:
It works well even with depth=0.
I compared output json in local without depth, depth=0, depth=1 and found no difference where as with depth=2 we have stop_area displayed.
I also added a test with depth=0

Reply:
It's ugly, but we've already got a lot of those very same cases in the codebase: depth is unsigned here, and signed into `fill()`.
Better to make it explicit:
```suggestion
                pb_creator.fill(item.stop_points[i], stop_time->mutable_stop_point(), int(depth) - 1);
```
The finest thing being to have all being signed (as we test the sign to stop display).